unifi_console/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,48 @@
+name: Unifi_Console
+files:
+  - name: unifi_console.yaml
+    options:
+      - template: init_config
+        options:
+          - template: init_config/http
+          - template: init_config/default
+      - template: instances
+        options:
+          - name: url
+            required: true
+            description: The controler url.
+            value:
+              type: string
+              example: https://127.0.0.1:8443
+          - name: user
+            required: true
+            description: User name
+            value:
+              type: string
+          - name: pwd
+            secret: true
+            required: true
+            description: Password
+            value:
+              type: string
```

Comment:
These are already included in the http template as `username` and `password`

Reply:
I tried, but they get picked up by the http library for a standard username/password http authentication which doesn't work in this case

Reply:
Ok, in that case you should disable those config options from http